config/externalname.go

```diff
@@ -1030,7 +1030,7 @@ var ExternalNameConfigs = map[string]config.ExternalName{
 	// Cloudwatch Log Groups can be imported using the name
 	"aws_cloudwatch_log_group": config.NameAsIdentifier,
 	// CloudWatch Log Metric Filter can be imported using the log_group_name:name
```

Comment:
Is this comment not an accurate description of the terraform id? It would be great to keep it up to date as well as the actual code.